rfcs/0022-transaction-structure/0022-transaction-structure.md

```diff
@@ -128,10 +128,14 @@ If you want to use a script in CKB, follow the code locating rules:
 
 - Compile your code into RISC-V binary. You can find some examples in the [repository](https://github.com/nervosnetwork/ckb-system-scripts) which builds the code for system cells.
 - Create a cell which stores the binary as data in a transaction, and send the transaction to the chain.
-- Construct a script structure, which `hash_type` is "Data", and `code_hash` is just the hash of the built binary.
+- Construct a script structure, which `hash_type` is "Data"[^1], and `code_hash` is just the hash of the built binary.
 - Use the script as the type or the lock script in a cell.
 - If the script has to run in a transaction, include the code cell's out point in the `cell_deps`.
 
+[^1]: Or "Data1" after [rfc32] is activated.
```

Comment:
This reminds me of something I've long wanted to ask: how do we do proper references in RFC? Should we just inline the link, or use a more traditional reference style footnotes at the end of the document?

Personally I feel using inline links work better since we are writing markdowns anyway, there are not so many chances where one wants to print the RFCs. But this is clearly bigger than one's preference, so the real question is: is there a decided style on references?